### ISSUE TITLE
Add a language / translate thing

### ISSUE THREAD

**Issue by Maingron** [Maingron/MainOS#23]
There should be a engine or variable that contains most common phrases like "Okay", "Yes", "No", "Abort", "Error", "Warning" in multiple languages. Maybe such phrases in programs could also be automatically replaced and translated onload.

**Comment by Maingron** [comment]
This includes a setting containing all languages which programs can ask for so that they can be translated by the developer.
